src/App.css

```diff
@@ -13,10 +17,12 @@
   margin-bottom: 2rem;
 }
 .article-card-img {
-  width: 10rem;
-  height: 10rem;
+  width: 100%;
```

Comment:
Really like the styling you've setup for the articles cards, they're perfectly in line on my screen!